pkg/scheduler/extender/extender.go

```diff
@@ -431,6 +445,42 @@ func (e *Extender) createReservation(ctx context.Context, namespace string, name
 		// cannot create reservation
 		return err
 	}
+	// wait for reservation to be confirmed
+	// continue on reading error
+	// return after reservation confirmed
+	// or wait context deadline
+	readCh := make(chan struct{})
+	ticker := time.NewTicker(time.Second)
```

Comment:
How many attempts are needed for one filter request? Should we make this timeout higher or increase in some time to not load k8s controller on large clusters?

Reply:
> How many attempts are needed for one filter request?

It depends

> Should we make this timeout higher or increase in some time to not load k8s controller on large clusters?

How much ?

Reply:
Not sure, need to have experiments. Could we have 10-20 sec timeout after some attempts?

Reply:
Where this timeouts (10-20 sec) comes from ?

Reply:
When I have increased RateLimiter Timeout, I made an experiment that `CreateVolume` request for a pod with 4 LVG volumes last 12.5 seconds, so the next ACR can became RESERVED after this period. It is applicable for baremetal Openshift cluster

Reply:
To make decision which timeout is most suitable we need better evidence(Tests and benchmarks with different cluster configurations and profiles.). I chose one second as start point.

Reply:
I think, it might be configurable to avoid this situation

Reply:
I don't know what we should denote in this issue. Any thoughts?‎

Reply:
I don't think its matter because context timeout is 15 seconds.

Reply:
Agree that we need to have exponential back-off here to avoid Kube API overloading

Reply:
ok

Reply:
I thought the deadline timeout is 5 minutes... If it's just 15 seconds, it may be not critical